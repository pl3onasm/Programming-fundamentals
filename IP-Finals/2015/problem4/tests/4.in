50
45
12
34
65
45
34
23
55
66
8
7
5
4
6
5
4
5
7
8
89
6
2
3
5
4
3
2
3
6
8
7
6
5
6
7
88
9
34
45
8
9
12
23
44
6
98
7
6
5
23