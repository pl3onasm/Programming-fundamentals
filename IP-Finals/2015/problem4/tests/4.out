22,27,23,26,28,13,16,25,12,15,17,24,33,49,14,21,29,32,34,45,48,11,18,31,35,47,10,19,30,40,37,41,2,42,7,43,50,3,6,38,44,1,5,39,8,4,9,36,20,46
